javaslang/generator/Generator.scala

```diff
@@ -448,6 +450,102 @@ def generateMainClasses(): Unit = {
           ${genTraversableAliases(ListType, "Seq")}
 
           ${genTraversableAliases(VectorType, "IndexedSeq")}
+
+          /$javadoc
+           * Alias for {@link $TreeSetType#of(Comparable)}
+           *
+           * @param <T>     Component type of element.
+           * @param element An element.
+           * @return A new $TreeSetType instance containing the given element
+           */
+          public static <T extends Comparable<? super T>> $TreeSetType<T> SortedSet(T element) {
+              return $TreeSetType.of(element);
+          }
+
+          /$javadoc
+           * Alias for {@link $TreeSetType#of($JavaComparatorType, Object)}
+           *
+           * @param <T>     Component type of element.
+           * @param comparator The comparator used to sort the elements
+           * @param element An element.
+           * @return A new $TreeSetType instance containing the given element
+           */
+          public static <T> $TreeSetType<T> SortedSet($JavaComparatorType<? super T> comparator, T element) {
+              return $TreeSetType.of(comparator, element);
+          }
+
+          /$javadoc
+           * Alias for {@link $TreeSetType#of(Comparable...)}
+           *
+           * @param <T>      Component type of element.
+           * @param elements Zero of more elements.
```

Comment:
`or` (note, I don't mind if you put these in a separate commit, as I know it can be a lot of work to rebase everything)